worklets/Overview.bs

```diff
@@ -410,7 +412,33 @@ loops and callbacks exceeding imposed time limits.
 Security Considerations {#security-considerations}
 ==================================================
 
-Issue(w3c/css-houdini-drafts#92): Need to decide if to allow worklets for unsecure context, etc.
+Worklets do not introduce any usable API surface to the web themselves, instead other
+specifications, e.g. [[css-paint-api-1]], [[webaudio]] will extend this specification to provide
+useful functionality.
+
+Worklets are allowed to be used outside <a>SecureContexts</a> as specifications which extend/use
+worklets may be allowed to used outside <a>SecureContexts</a>.
+
+Worklets load their module scripts in the same manner that workers load their module scripts. This
+fetches the top-level script by the <a>fetch a single module script</a> algorithm which sets the
+<a for=request>mode</a> to "<code>cors</code>".
+
+The behaviour of loading scripts can be restricted by using <a>child-src</a> [[CSP]] directive. Each
+specification which extends this specification <em>must</em> provide a <a
+for=request>destination</a> which is sent along with requests in the <a>Fetch a module worker script
+graph</a> algorithm.
```

Comment:
Actually, worklets should be guarded by the script directive. But also, I'm not sure that needs to be mentioned in this section.